packages/tectonic/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://tectonic-typesetting.github.io/
+TERMUX_PKG_DESCRIPTION="Tectonic is a modernized, complete, self-contained TeX/LaTeX engine, powered by XeTeX and TeXLive."
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="Naveed Pasha <naveedpash@gmail.com>"
+TERMUX_PKG_VERSION=tectonic@0.4.1
+TERMUX_PKG_GIT_BRANCH=tectonic@0.4.1
+TERMUX_PKG_SRCURL=https://github.com/tectonic-typesetting/tectonic.git
+TERMUX_PKG_SHA256=5a2c910f822d59ddaf9d32a0e5f7f34ce30f44e4129513b3a0c50425cf48ac8f
+TERMUX_PKG_DEPENDS="harfbuzz"
```

Comment:
Yes true but the termux `harfbuzz` package already depends on the exact same dependencies mentioned on their docs so the build system built them as part of the process. Should I still list those dependencies in `TERMUX_PKG_DEPENDS` so that the appropriate *.debs will be downloaded?

Reply:
I think it would be good, it makes it easier to maintain the program. It seem to for example depend on openssl. When a major upgrade of openssl is released all packages depending on it needs to be rebuilt, if openssl is not listed in TERMUX_PKG_DEPENDS we will probably miss to rebuild it